source/aws-bootstrap-kit/lib/dns.ts

```diff
@@ -42,6 +48,7 @@ export class RootDns extends Construct {
         account,
         props.rootHostedZoneDNSName
       );
+      this.stagesHostedZones.push(stageSubZone);
```

Comment:
only for tests?

Reply:
What do you mean ? just populating the exposed array for potential reused by customer

Reply:
OK